gnome-initial-setup/pages/language/cc-language-chooser.c

```diff
@@ -415,6 +416,8 @@ sort_languages (GtkListBoxRow *a,
                 GtkListBoxRow *b,
                 gpointer       data)
 {
+        CcLanguageChooser *chooser = data;
+        CcLanguageChooserPrivate *priv = cc_language_chooser_get_instance_private (chooser);
         LanguageWidget *la, *lb;
         gchar *normalized_a, *normalized_b;
         gint retval;
```

Comment:
Also not in the new code, but it appears that the 'retval' variable is never used.

Reply:
Ignore that, too.  Again, my error in not expanding all the code when reviewing in github.